src/App.js

```diff
@@ -1,11 +1,18 @@
-import './scss/App.scss';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Books from './components/Books';
+import Categories from './components/Categories';
 
-function App() {
-  return (
-    <>
-      <h1 className="text-8xl font-bold text-red-500 text-center">Works</h1>
-    </>
-  );
-}
+const App = () => (
+  <>
+    <Header />
+    <Routes>
+      <Route path="/" element={<Books />} />
```

Comment:
- [ ] Please, consider that in order to **not** show a blank page when visiting the site for the first time, the `Books` component should be the default page - so, you should add `exact` before the path, like `<Route exact path="/">` 👍